docs/src/dev/provider/gremlin-semantics.asciidoc

```diff
@@ -618,24 +618,26 @@ link:https://tinkerpop.apache.org/docs/x.y.z/reference/#asDate-step[reference]
 [[asString-step]]
 === asString()
 
-*Description:* Returns the value of incoming traverser as strings
+*Description:* Returns the value of incoming traverser as strings, or if `Scope.local` is specified, returns each element inside
+incoming list traverser as string.
 
-*Syntax:* `asString()`
+*Syntax:* `asString()` | `asString(local)`
```

Comment:
This looks a bit odd to me. I'd think it would be something like `asString(Scope)`

Reply:
Yea I think there were a few misses on this doc...I'll scan through and update all mistakes on this.